pkg/model/components/etcdmanager/model.go

```diff
@@ -342,6 +342,9 @@ func (b *EtcdManagerBuilder) buildPod(etcdCluster *kops.EtcdClusterSpec) (*v1.Po
 				awsup.TagNameRolePrefix + "master=1",
 			}
 			config.VolumeNameTag = awsup.TagNameEtcdClusterPrefix + etcdCluster.Name
+			container.Env = []v1.EnvVar{
+				{Name: "AWS_REGION", Value: os.Getenv("AWS_REGION")},
```

Comment:
Is AWS_REGION set to the state store bucket?  I thought we decided it wasn't necessarily the case.

I think we need to explicitly figure out the region of the bucket, and pass that in.

Reply:
@justinsb If we could infer the region of the bucket that would be best, but the path to the state store bucket is often written in `S3Uri`, like `s3://kops-state-store`, one may not figure out the region easily.